src/main/java/com/owczarczak/footballers/clubRepresentation/ClubRepresentationController.java

```diff
@@ -0,0 +1,39 @@
+package com.owczarczak.footballers.clubRepresentation;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+import static org.springframework.http.ResponseEntity.notFound;
+import static org.springframework.http.ResponseEntity.ok;
+
+@RestController
+@RequestMapping("/representation")
+public class ClubRepresentationController {
+    @Autowired
+    ClubRepresentationService service;
```

Comment:
is this meant for access by other classes in the same package or subclasses?

Reply:
It's meant to be accessed only in ClubRepresentationController class.
Changed to private.